desktop/core/src/desktop/urls.py

```diff
@@ -202,6 +204,13 @@
 ]
 
 dynamic_patterns += [
+    re_path('api/schema',
+        SpectacularAPIView.as_view(permission_classes=(AllowAny,)),
+        name='schema'),
+  re_path('api/swagger',
+        SpectacularSwaggerView.as_view(url_name='schema', permission_classes=(AllowAny,)),
+        name='swagger-ui'),
+
```

Comment:
Missing redoc mapping?

Reply:
no intentionally removed it,  as we have swagger. we can add it if required.